ajax/libs/exceljs/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "exceljs",
+  "version": "0.8.1",
+  "filename": "exceljs.min.js",
+  "description": "Excel Workbook Manager - Read and Write xlsx and csv Files.",
+  "license": "MIT",
+  "author": {
+    "name": "Guyon Roche"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/guyonroche/exceljs.git"
+  },
+  "keywords": [
+    "xlsx",
+    "json",
+    "csv",
+    "excel",
+    "font",
+    "border",
+    "fill",
+    "number",
+    "format",
+    "number format",
+    "alignment",
+    "office",
+    "spreadsheet",
+    "workbook",
+    "defined names",
+    "data validations",
+    "rich text",
+    "in-cell format",
+    "outlineLevel",
+    "views",
+    "frozen",
+    "split",
+    "pageSetup"
+  ],
+  "npmName": "exceljs",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "exceljs*",
```

Comment:
Let's use one simple rule `!(es5|es3){,**/*}` to fetch files we need! :smile: 
It means we want to fetch all files under dist except `es5` or `es3` themself and sub-folders and files inside.
`es3` is for v0.4.5 & v0.4.6.
Sorry for not figuring out this one previously.

Reply:
Thanks.